src/build-data/os/darwin.txt

```diff
@@ -21,6 +21,8 @@ commoncrypto
 sockets
 threads
 filesystem
+
+certstore
```

Comment:
Having this feature named certstore is going to get confusing since Windows etc also has a certificate store. Maybe something like `apple_keychain`?

Reply:
Judging from "filesystem", "threads" and so on, I assumed that this is a list of "features" a certain platform provides. I.e. if we'd add a similar implementation for Windows (which we are actually planning in the near future), we would just add "certstore" to `windows.txt` as well.
Is that a wrong assumption?

Reply:
Here feature is in terms of specific APIs, so for `threads` we mean that `std::thread` and company work, and `sockets` means BSD socket API (which is why Windows defines `winsock2` instead as the APIs differ). If say there was a platform which had threads but the `std::` thread API didn't work and we had to use some system specific API, it would get its own distinct feature and not define `threads`.

Reply:
That's good to know, thanks.